pkg/repository/notificationrepository/notification_db_models.go

```diff
@@ -27,6 +27,17 @@ type notificationRow struct {
 	CustomFields []byte  `db:"custom_fields"`
 }
 
+func notificationFieldMapping() map[string]string {
+	return map[string]string{
+		dtos.NameField:            "title",
+		dtos.DescriptionFieldName: "detail",
+		dtos.LevelFieldName:       "level",
+		dtos.OccurrenceFieldName:  "timestamp",
+		dtos.OriginFieldName:      "origin",
+		dtos.TitleFieldName:       "title",
```

Comment:
are there intentionally two entries mapping to `title` ?

Reply:
The title filter field has been removed.